add model example in tags doc (#6757)
dbt-labs/docs.getdbt.com: website/docs/reference/resource-configs/tags.md

```

</File>

> For example, to apply tags to a model in your `models/` directory, you would add the following using the `config` property:

Reviewer: ```suggestion
To apply tags to a model in your `models/` directory, add the `config` property similar to the following example:
```

- `dbt run --select +model_name+` &mdash; Run a model, its upstream dependencies, and its downstream dependencies.
- `dbt run --select tag:my_tag+ --exclude tag:exclude_tag` &mdash; Run model tagged with `my_tag` and their downstream dependencies, and exclude models tagged with `exclude_tag`, regardless of their dependencies.

:::tip Usage notes about tags

> 

Reviewer: Always feels a little bit weird to me to jump into bullets with no lead in. How about something like:
```suggestion

When using tags, consider the following: 

```

        - "published"

```
</File>

### Apply tags to models

> You can also apply tags to individual resources, like models, using in the `dbt_project.yml`, `schema.yml`, or SQL files. 

Author: ```suggestion
You can also apply tags to models using in the `dbt_project.yml`, `schema.yml`, or SQL files. You can apply tags to individual resources as well.
```

> You can also apply tags to individual resources, like models, using in the `dbt_project.yml`, `schema.yml`, or SQL files. 

Author: ```suggestion
This section shares how to apply tags to models in the `dbt_project.yml`, `schema.yml`, or SQL files, however you can also apply tags to individual resources. 
```

> This section shares how to apply tags to models in the `dbt_project.yml`, `schema.yml`, or SQL files, however you can also apply tags to individual resources. 

Reviewer: I'd leave that section out if we've removed that part of this section or link to wherever else it is
```suggestion
This section demonstrates applying tags to models in the `dbt_project.yml`, `schema.yml`, and SQL files. 
```
